packages/calendar/src/LionCalendar.js

```diff
@@ -22,50 +22,7 @@ import { createDay } from './utils/createDay.js';
 export class LionCalendar extends LocalizeMixin(LitElement) {
   static get localizeNamespaces() {
     return [
-      {
-        'lion-calendar': locale => {
-          switch (locale) {
-            case 'bg-BG':
-              return import('../translations/bg.js');
-            case 'cs-CZ':
-              return import('../translations/cs.js');
-            case 'de-AT':
-            case 'de-DE':
-              return import('../translations/de.js');
-            case 'en-AU':
-            case 'en-GB':
-            case 'en-PH':
-            case 'en-US':
-              return import('../translations/en.js');
```

Comment:
Also note that this isn't exact refactoring because currently multiple locales are targeting same file in some cases.